switchboard/switchboard/src/bin/swx.rs

```diff
@@ -0,0 +1,95 @@
+//! The actual switchboard runner. Run as a command-line tool.
+
+use clap::{Parser, Subcommand};
+use std::path::PathBuf;
+use std::time::Duration;
+use uuid::Uuid;
+
+/// Arguments passed to the switchboard from the command line
+#[derive(Debug, Parser)]
+#[command(version, about)]
+pub struct Args {
+    #[command(subcommand)]
+    pub command: Command,
+}
+#[derive(Debug, Subcommand)]
+#[command(about)]
+pub enum Command {
+    /// Run the switchboard server.
+    Serve {
+        /// Path to server configuration file
+        #[arg(short = 'c', long = "cfg", env = "TML_CFG")]
+        cfg: PathBuf,
+    },
```

Comment:
```suggestion
    Serve(tml_switchboard::server::serve::ServeConfig),
```

And then define the `ServeConfig` struct analog to the above:
```rust
struct ServeConfig {
    /// Path to server configuration file
    #[arg(short = 'c', long = "cfg", env = "TML_CFG")]
    cfg: PathBuf,
},
```

This would increase cohesion within the implementation of these (independent) subcommands and reduce coupling between the `swx` tool and the respective subcommands.